Content.Server/Construction/AnchorableSystem.cs

```diff
@@ -85,7 +89,26 @@ private void OnAnchorComplete(EntityUid uid, AnchorableComponent component, TryA
             // TODO: Anchoring snaps rn anyway!
             if (component.Snap)
             {
-                _transform.SetCoordinates(uid, xform.Coordinates.SnapToGrid(EntityManager, _mapManager));
+                var coordinates = xform.Coordinates.SnapToGrid(EntityManager, _mapManager);
+
+                if (_tagSystem.HasTag(uid, "Unstackable"))
+                {
+                    // Need to enforce the unstackable rules on anchoring also.
+                    var condition = new NoUnstackableInTile();
```

Comment:
Kinda weird to allocate the class for it, probably better as just a method call, it also doesn't get checked when the anchoring is started it looks like?

Reply:
Well, there is no allocation since it's an empty class. I'd have to make that method static to reuse it without a new.

Reply:
I introduced a static method for this.